packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8731,627 +8782,7 @@ describe('plugin-meetings', () => {
-            requiredHints: [DISPLAY_HINTS.DISABLE_SHOW_DISPLAY_NAME],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_CONTROL],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.ENABLE_VIEW_THE_PARTICIPANT_LIST],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.DISABLE_VIEW_THE_PARTICIPANT_LIST],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_FILE],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasPolicies, {
-            requiredPolicies: [SELF_POLICY.SUPPORT_FILE_SHARE],
-            policies: selfUserPolicies,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_APPLICATION],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasPolicies, {
-            requiredPolicies: [SELF_POLICY.SUPPORT_APP_SHARE],
-            policies: selfUserPolicies,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_CAMERA],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasPolicies, {
-            requiredPolicies: [SELF_POLICY.SUPPORT_CAMERA_SHARE],
-            policies: selfUserPolicies,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_DESKTOP],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasPolicies, {
-            requiredPolicies: [SELF_POLICY.SUPPORT_DESKTOP_SHARE],
-            policies: selfUserPolicies,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasHints, {
-            requiredHints: [DISPLAY_HINTS.SHARE_CONTENT],
-            displayHints: userDisplayHints,
-          });
-          assert.calledWith(ControlsOptionsUtil.hasPolicies, {
-            requiredPolicies: [SELF_POLICY.SUPPORT_VOIP],
-            policies: selfUserPolicies,
-          });
-
-          assert.calledWith(updateMeetingActionsSpy);
-          assert.calledWith(setRecordingDisplayHintsSpy, userDisplayHints);
-          assert.calledWith(setUserPolicySpy, userDisplayPolicy);
-          assert.calledWith(setControlsDisplayHintsSpy, userDisplayHints);
-          assert.calledWith(handleDataChannelUrlChangeSpy, datachannelUrl);
-        });
-      });
-
-      describe('#updateMeetingActions', () => {
```

Comment:
Are these tests no longer necessary? Why are they removed?

Reply:
These lines are duplicate. Checked in beta branch as well these tests are only present once, we might have introduced this while merging the branches